src/DssSpell.sol

```diff
@@ -73,189 +53,105 @@ contract DssSpellAction is DssAction {
-            OPERATOR: 0x8Fe38D1E4293181273E2e323e4c16e0D1d4861e3,
-            INPUT_CONDUIT: 0x8Fe38D1E4293181273E2e323e4c16e0D1d4861e3,
-            OUTPUT_CONDUIT: 0x8Fe38D1E4293181273E2e323e4c16e0D1d4861e3,
-            URN:  0x0C185bf5388DdfDB288F4D875265d456D18FD9Cb,
-            gemID: "RWA006",
-            joinID: "MCD_JOIN_RWA006_A",
-            urnID: "RWA006_A_URN",
-            inputConduitID: "RWA006_A_INPUT_CONDUIT",
-            outputConduitID: "RWA006_A_OUTPUT_CONDUIT",
-            pipID: "PIP_RWA006",
-            ilk: "RWA006-A",
-            ilk_string: "RWA006-A",
-            ilkRegistryName: "RWA006-A: Centrifuge: Alternative Equity Advisers",
-            RATE: TWO_PCT,
-            CEIL: 0 * MILLION,
-            PRICE: 0 * WAD,
-            MAT: 10_000,
-            TAU: 0,
-            DOC: ""
-        });
+    function actions() public override {
+        address MCD_VAT  = DssExecLib.vat();
 
-        CentrifugeCollateralValues[4] memory collaterals = [RWA003, RWA004, RWA005, RWA006];
+        // -----------  Core Unit Budget Payouts - August -----------
 
-        // Integrate RWA003-006
-        for (uint i = 0; i < collaterals.length; i++) {
-            integrateCentrifugeCollateral(collaterals[i]);
-        }
+        DssExecLib.sendPaymentFromSurplusBuffer(GRO_MULTISIG,    637_900);
+        DssExecLib.sendPaymentFromSurplusBuffer(SES_MULTISIG,    702_883);
+        DssExecLib.sendPaymentFromSurplusBuffer(MKT_MULTISIG,    44_375);
+        DssExecLib.sendPaymentFromSurplusBuffer(GOV_MULTISIG,    273_334);
+        DssExecLib.sendPaymentFromSurplusBuffer(RWF_MULTISIG,    155_000);
+        DssExecLib.sendPaymentFromSurplusBuffer(RISK_CU_EOA,     182_000);
+        DssExecLib.sendPaymentFromSurplusBuffer(PE_MULTISIG,     510_000);
+        DssExecLib.sendPaymentFromSurplusBuffer(ORA_MULTISIG,    419_677);
+        DssExecLib.sendPaymentFromSurplusBuffer(COM_MULTISIG,    40_500);
+        DssExecLib.sendPaymentFromSurplusBuffer(COM_ER_MULTISIG, 121_500);
+        //                                                     _________
+        //                                         TOTAL DAI:  3,087,169
 
-        // Bump changelog version
-        DssExecLib.setChangelogVersion("1.9.2");
-    }
+        // ----------- Maker Open Market Commitee Proposal -----------
+        // https://vote.makerdao.com/polling/QmVG38FK?network=mainnet#poll-detail
 
-    function integrateCentrifugeCollateral(CentrifugeCollateralValues memory collateral) internal {
-        address MIP21_LIQUIDATION_ORACLE =
-            DssExecLib.getChangelogAddress("MIP21_LIQUIDATION_ORACLE");
+        // ETH-B Stability Fee Decrease 6% to 5%
+        DssExecLib.setIlkStabilityFee("ETH-B", FIVE_PCT, true);
 
-        address vat = DssExecLib.vat();
+        // Maximum Debt Ceiling Decreases.
+        DssExecLib.setIlkAutoLineDebtCeiling("LRC-A", 1 * MILLION); // Decrease 3 million to 1 million.
```

Comment:
Okay, here are my thoughts.  I think this could cause our global DC decrease problem when we limit ceilings.  The good news is, this collateral is only using 879k, so unless someone consumes the DC, we are probably fine.  In the future though, if we wanted to limit a collateral with the autoline, and that collateral's debt is over where we are limiting it, the autoline will reduce the global DC potentially preventing other collateral types from issuing new debt.

I think the overall magnitudes and situation with `LRC-A` means this is a non-issue.  Even if we were at the DC with `LRC-A`, it would only mean 2 million in the global DC that would count against us.  The question is, even for this small amount, should we build a pattern like this is a larger amount that would have the problem?  That way, when one looks at the archive to see how to do this, the pattern is correct.

The "right" solution is to get the `LRC-A`'s actual debt, compute the difference between the new `line` and the debt.  If real debt is lower than the new `line`, then we're good and the autoline can just be adjusted down, otherwise we need to adjust the autoline down, but then also add the delta to the global DC.  Second opinions, or does someone see a more elegant solution?  The right way to fix this might be to put this logic in the `DssExecLIb`.

Reply:
For context, here is what we did wrong last time:
https://github.com/makerdao/spells-mainnet/blob/master/archive/2021-05-14-DssSpell.sol#L103-L110

Reply:
I don't favor putting the extra logic into this spell--there's no way to enforce that people go back and find examples. Having an exec lib function like `setCollateralShutdownCeiling` is probably the reasonable solution, although we're in jeopardy of having to upgrade the exec lib if we change the procedure.

Reply:
Another possible concern is that the difference between the real debt and the newLine can be increased even *after* changing the ilk's line in the auto-line because the changes aren't reflected in the `Vat`'s ilk-specific line, nor the Line, until `exec` is called on the auto-line & the DssExecLib's function to adjust auto-line does not call `exec` - but it can, since it decreases in the line can be performed regardless of `ilk.lastInc`'s value. 

I know there are services `exec` regularly, but as far as I can tell it isn't called in the spell, and the adjustment of the line downwards should be an atomic operation that's also reflected in the `Vat`.

Reply:
We decided that we are going to make a ticket to update the `DssExecLib` out of this discussion.

Reply:
created a ticket: https://github.com/makerdao/smart-contracts-team-pm/issues/635